bootstrap/bootstrap.go

```diff
@@ -163,6 +163,26 @@ func (b *Bootstrap) StartTraceDownloader(ctx context.Context) error {
 	)
 
 	StartEngine(ctx, b.traces, l)
+
+	if b.config.TracesBackfillStartHeight > 0 {
```

Comment:
How do we prevent from backfilling it over and over again after every restart?

Reply:
That can be done by completely removing the provided values for these 2 flags, upon restarting, or simply by shifting their values to not account for past heights, which we know were back-filled.

Reply:
Yeah, it's better that we could somehow remember that we have back-filled, and skip it next time.

Reply:
That's true, but soon enough the traces will be generated on demand from the EVM Gateway, so there will be no need for downloading from a GCP bucket. I see this merely as a band-aid to unblock partners that rely on the missing traces.

Reply:
I was thinking we would just remove the flag from the config after starting it up. We could build a more intelligent solution, but I think this is going away in the next couple months so not worth spending much time on it.

worst case it will scan for all of the traces, and skip downloading them because they already exist in the db